panic: runtime error: invalid memory address or nil pointer dereference

version: 3.5.1

after running successfully in no-dry-run a few times all the way to completion, randomly received the following SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x43557ff on an additional run. Ran again without receiving it.

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x43557ff]

goroutine 1 [running]:
main.main.func1()
	github.com/ekristen/aws-nuke/v3/main.go:26 +0x54
panic({0x52a3820?, 0xa7d6a30?})
	runtime/panic.go:914 +0x21f
github.com/ekristen/aws-nuke/v3/resources.(*RDSSnapshot).Properties(0xc002baa1b0)
	github.com/ekristen/aws-nuke/v3/resources/rds-snapshots.go:103 +0x13f
github.com/ekristen/libnuke/pkg/queue.(*Item).GetProperty(0xc000afb860, {0xc0001f9400, 0x9})
	github.com/ekristen/libnuke@v0.18.0/pkg/queue/item.go:76 +0x12e
github.com/ekristen/libnuke/pkg/nuke.(*Nuke).Filter(0xc0004b21e0, 0xc000afb860)
	github.com/ekristen/libnuke@v0.18.0/pkg/nuke/nuke.go:465 +0xa9b
github.com/ekristen/libnuke/pkg/nuke.(*Nuke).runScanner(0xc0004b21e0, {0x6ca4478?, 0xc0000ad0e0?}, 0xc0000cd920, 0xc0007a0000)
	github.com/ekristen/libnuke@v0.18.0/pkg/nuke/nuke.go:387 +0x245
github.com/ekristen/libnuke/pkg/nuke.(*Nuke).Scan(0xc0004b21e0, {0x6ca4478, 0xc0000ad0e0})
	github.com/ekristen/libnuke@v0.18.0/pkg/nuke/nuke.go:412 +0x1b6
github.com/ekristen/libnuke/pkg/nuke.(*Nuke).Run(0xc0004b21e0, {0x6ca4478, 0xc0000ad0e0})
	github.com/ekristen/libnuke@v0.18.0/pkg/nuke/nuke.go:198 +0xda
github.com/ekristen/aws-nuke/v3/pkg/commands/nuke.execute(0xc000050280)
	github.com/ekristen/aws-nuke/v3/pkg/commands/nuke/nuke.go:205 +0x180a
github.com/urfave/cli/v2.(*Command).Run(0xc000502000, 0xc000050280, {0xc0001c51e0, 0xb, 0xb})
	github.com/urfave/cli/v2@v2.27.2/command.go:276 +0x9dd
github.com/urfave/cli/v2.(*Command).Run(0xc0005026e0, 0xc000050140, {0xc0001b4000, 0xc, 0xc})
	github.com/urfave/cli/v2@v2.27.2/command.go:269 +0xc2e
github.com/urfave/cli/v2.(*App).RunContext(0xc0000be200, {0x6ca4210?, 0xa818e40}, {0xc0001b4000, 0xc, 0xc})
	github.com/urfave/cli/v2@v2.27.2/app.go:333 +0x5db
github.com/urfave/cli/v2.(*App).Run(...)
	github.com/urfave/cli/v2@v2.27.2/app.go:307
main.main()
	github.com/ekristen/aws-nuke/v3/main.go:46 +0x1eb
```

--- a/resources/rds-snapshots.go
+++ b/resources/rds-snapshots.go
@@ -99,8 +99,11 @@ func (i *RDSSnapshot) Properties() types.Properties {
 		Set("Identifier", i.snapshot.DBSnapshotIdentifier).
 		Set("SnapshotType", i.snapshot.SnapshotType).
 		Set("Status", i.snapshot.Status).
-		Set("AvailabilityZone", i.snapshot.AvailabilityZone).
-		Set("CreatedTime", i.snapshot.SnapshotCreateTime.Format(time.RFC3339))
+		Set("AvailabilityZone", i.snapshot.AvailabilityZone)
+
+	if i.snapshot != nil && i.snapshot.SnapshotCreateTime != nil {
+		properties.Set("CreatedTime", i.snapshot.SnapshotCreateTime.Format(time.RFC3339))
+	}
 
 	for _, tag := range i.tags {
 		properties.SetTag(tag.Key, tag.Value)

